File: threads/cpp/qt/threadb.cpp
#include "threadb.h"
#include <QDebug>

ThreadB::ThreadB(QObject *parent) :
	QThread(parent)
{
}

void ThreadB::run()
{
	for(int i=0; i<10; i++)
	{
		qDebug("B");
		sleep(1);
	}
}

File: threads/java/src/threads/ThreadB.java
package threads;

public class ThreadB extends Thread {
	@Override
	public void run() {
		for(int i=0; i<10; i++)
		{
			try {
				sleep(500);
		    }
		    catch(InterruptedException e) 
		    {
		    }
			
			System.out.println("B");
		}
	}
}
